llvm/include/llvm/IR/CallingConv.h

```diff
@@ -261,6 +261,13 @@ namespace CallingConv {
     /// except that the first parameter is mapped to x9.
     ARM64EC_Thunk_Native = 109,
 
+    // Calling convention used by STM8 ABI according to SDCC STM8
+    // calling conv version 1
+    STM8_BUILTIN = 110,
+
+    // STM8 Interrupt calling conv
+    STM8_INTR = 111,
```

Comment:
Is the interrupt calling convention documented somewhere?

Reply:
Not necessarily. But it is still different (e.g. no return). I think we will find more regarding this by looking at the programming manual by ST